### PR TITLE
[WIP] feature/47975

### PR DESCRIPTION
**Pull request by robwozniak**

### CODE REVIEW

**Comment by szafran89** [comment]
Please set proper configuration b/c now preview looks like this http://prntscr.com/lqm2pp

File: components/01-globals/grid/grid.config.js [hunk 1]
```diff
@@ -1,23 +1,239 @@
...
+          ['col-xs-offset-9 col-xs-3'],
+          ['col-xs-offset-8 col-xs-4'],
+          ['col-xs-offset-7 col-xs-5'],
+          ['col-xs-offset-6 col-xs-6'],
+          ['col-xs-offset-5 col-xs-7'],
+          ['col-xs-offset-4 col-xs-8'],
+          ['col-xs-offset-3 col-xs-9'],
+          ['col-xs-offset-2 col-xs-10'],
+          ['col-xs-offset-1 col-xs-11']
+        ]
```
Comment (szafran89) [inline components/01-globals/grid/grid.config.js]: I removed offsets from sass code, therefore we should remove this variant

File: components/01-globals/mixins/_grid-column-width.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @mixin grid-column-width($width, $count) {
-    @if ($grid__gutter > 0) {
-        flex-basis: calc(#{100% / $count * $width} - #{($count - $width) * $grid__gutter / $count});
-        max-width: calc(#{100% / $count * $width} - #{($count - $width) * $grid__gutter / $count});
+    @if ($grid__gutter-width > 0) {
+        flex-basis: calc(#{100% / $count * $width} - #{($count - $width) * $grid__gutter-width / $count});
+        max-width: calc(#{100% / $count * $width} - #{($count - $width) * $grid__gutter-width / $count});
     }
```
Comment (szafran89) [inline components/01-globals/mixins/_grid-column-width.scss]: can we remove this mixin at all?

File: components/04-views/cart/cart.hbs [hunk 3]
```diff
@@ -3,113 +3,121 @@
...
+                            class="
+                                row
+                                row--no-margins
+                                between-xs
+                                cart-totals__row
+                                cart-totals__row--summary
+                            "
                         >
-                            $34.00
+                            <div class="
```
Comment (szafran89) [inline components/04-views/cart/cart.hbs]: class to new line

File: components/04-views/cart/cart.hbs [hunk 4]
```diff
@@ -3,113 +3,121 @@
...
+                                    cart-totals__row-label
+                                    cart-totals__row-label--total
+                                "
+                            >
+                                Order Total
+                            </div>
+                            <div class="
```
Comment (szafran89) [inline components/04-views/cart/cart.hbs]: class to new line

File: components/01-globals/mixins/_grid-column-width.scss (hunk 2 above)
Comment (robwozniak) [reply components/01-globals/mixins/_grid-column-width.scss]: You're right, removed

**Pull request merged**